content/en/docs/configuration/acme/dns01/_index.md

```diff
@@ -86,22 +86,22 @@ By default, cert-manager will not follow CNAME records pointing to subdomains.
 
 If granting cert-manager access to the root DNS zone is not desired, then the
 `_acme-challenge.example.com` subdomain can instead be delegated to some other,
-less privileged domain. This could be achieved in the following way. Say, one has two zones:
+less privileged domain (`example.org`). This could be achieved in the following way. Say, one has two zones:
 
 * `example.com`
-* `less-privileged.com`
+* `example.org`
```

Comment:
How about `less-privileged.example.org`? is more clear to the user

Reply:
Okay, I can change it. My only concern is, if the 3rd level domain (`less-privileged.example.org`) next to the 2nd level domain (`example.com`) will not confuse a user with implication, that less-privileged domain = a 3rd level domain. I actually find the _less-privileged_ term used often in this section already confusing, as in reality, to achieve the DNS01 delegation any _other_ domain can be used, if you understand what I mean :smile: